common/src/main/res/values/strings.xml

```diff
@@ -56,6 +56,11 @@
     <string name="basic_sensor_name_bluetooth_ble_beacon_monitor">Beacon Monitor</string>
     <string name="basic_sensor_name_bluetooth_state">Bluetooth State</string>
     <string name="basic_sensor_name_bluetooth">Bluetooth Connection</string>
+    <string name="basic_sensor_name_car_battery_level">Battery Level</string>
```

Comment:
This string duplicates `basic_sensor_name_battery_level` at line 52 just above, we should either remove or rename it so its unique

Reply:
I renamed it to `Car Battery Level` so that it isn't confused with the phone battery level